Gulpfile.mjs

```diff
@@ -281,7 +279,7 @@ async function buildBabel(useWorker, ignore = []) {
     const dest = "./" + mapSrcToLib(file.slice(2));
     promises.push(
       worker.transform(file, dest, {
-        sourceMaps: enableSourceMap && !file.endsWith(".d.ts"),
+        sourceMaps: !file.endsWith(".d.ts"),
```

Comment:
In fact, since this environment variable was not set during the build process, we have carried source maps in past releases.